src/OnboardingSPA/store/reducer.js

```diff
@@ -193,6 +193,17 @@ export function data( state = {}, action ) {
 					},
 				},
 			};
+		case 'SET_SITEGEN_AI_ERROR_STATUS':
+			return {
+				...state,
+				flowData: {
+					...state.flowData,
+					sitegen: {
+						...state.flowData.sitegen,
+						siteGenErrorStatus: action.siteGenErrorStatus,
+					},
+				},
+			};
```

Comment:
We should also add this key to [Flows.php](https://github.com/newfold-labs/wp-module-onboarding-data/blob/9b97cc784db7d10afac694a7b065eb92d2ccf81c/includes/Flows/Flows.php#L140) and increment the flow data version.

Reply:
Done